src/Model/Build/BitbucketServerBuild.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+namespace PHPCensor\Model\Build;
+
+use GuzzleHttp\Client;
```

Comment:
@fejal Could you remove useless namespaces (`GuzzleHttp\Client`, `PHPCensor\Config`, `PHPCensor\Model\BuildError`) please?

Reply:
Absolutely.